docs/jsdoc/options.jsdoc

```diff
@@ -36,6 +36,10 @@
  * The filename of the template. Required for inclusion and caching unless 
  * you are using {@link module:ejs.renderFile}. Also used for error reporting.
  * 
+ * @property {String}  [root='undefined']
```

Comment:
Sorry about the confusion. Quotes should NOT be there in either case. I misread your comment before my reply.